test-runner-frontend/package.json

```diff
@@ -16,6 +17,7 @@
     "buffer": "^6.0.3",
     "firebase": "^9.9.1",
     "firebase-admin": "^11.0.1",
+    "jspdf-html2canvas": "^1.4.9",
```

Comment:
whats this dependency for?

Reply:
It's for downloading the Report View as PDF but we don't need it right now so I'll delete it.